service/raft/group0_state_machine.cc

```diff
@@ -26,6 +26,7 @@
 #include "idl/frozen_schema.dist.impl.hh"
 #include "idl/experimental/broadcast_tables_lang.dist.hh"
 #include "idl/experimental/broadcast_tables_lang.dist.impl.hh"
+#include "service/join_node.hh"
```

Comment:
Why do you need to add this here and in the files below in this patch?

Reply:
They aren't needed. I'll remove them. Perhaps I needed them in some previous version of the implementation, but I don't remember why.